rtengine/procparams.cc

```diff
@@ -8742,6 +8742,11 @@ int ProcParams::load(const Glib::ustring& fname, ParamsEdited* pedited)
                 assignFromKeyfile(keyFile, "Locallab", "ShapeMethod_" + index_str, spot.shapeMethod, spotEdited.shapeMethod);
                 if (keyFile.has_key("Locallab", "AvoidgamutMethod_" + index_str)) {
                     assignFromKeyfile(keyFile, "Locallab", "AvoidgamutMethod_" + index_str, spot.avoidgamutMethod, spotEdited.avoidgamutMethod);
+                    if (ppVersion < 351) {
+                       if(spot.avoidgamutMethod == "XYZ") {//5.10 default value
+                           spot.avoidgamutMethod = "MUNS";//set to Munsell only
+                       }
+                    }
```

Comment:
XYZ is a valid choice, so I think the method should not be changed.

Reply:
Excuse my bad english but I don't understand...

Yes "XYZ" (XYZ absolute) - it's now 5.10 default -  is a valid choice, but in some cases it leads to artifacts.

Rearranging the "clips" and choosing "Munsell only" is I think in 95% of cases a better (the best?) solution.

Jacques

Reply:
Even if XYZ causes artifacts in some cases, it is not a good idea for us to change parameters of existing edits. Those edits have been made while taking into account the effect of the avoid color shift method. Some people may have explicitly set the method to XYZ. The only time we should change edits is if it is a bug-fix.

Reply:
OK 

Jacques